### PR TITLE
Create Chinese video ad filtering

### PR DESCRIPTION
**Pull request by snforu**
[//]: # (***You can delete or ignore strings starting with "[//]:" They will not be visible either way.)

***Description***:
* **Current behaviour**: 

[//]: # (Substitute this line with a description of the problem)

[//]: # (Replace %screenshot_url% below with a link to the screenshot of the problem. Also, you can paste image from clipboard instead. It will be automatically loaded.)
<details><summary>Screenshot:</summary>

image
</details><br/>

* **Expected behaviour**: 

[//]: # (Substitute this line with a description of what should happen normally. If needed, provide a screenshot below, same as above)

<details><summary>Screenshot:</summary>

image
</details><br/>

***Steps to reproduce the problem***:

[//]: # (Substitute this line with a step-by-step instruction how to reproduce the issue)

***System configuration***

**Filters:**

[//]: # (Substitute this line with the list of your active filters, separated by commas)

[//]: # (Please enter the correct values for your case to the table below)

Information                            | Value
---                                    | ---
Operating system:                      | ?
Operating system version (Android/iOS) | ?
Browser:                               | ?
AdGuard version:                       | ?
Filters enabled:                       | ?
AdGuard mode (Android only):           | VPN / Proxy (manual/auto)
Filtering quality (Android only):      | High-quality / High-speed / Simplified
HTTPS filtering (Android only):        | On (Default/Blacklist mode) / Off
Simplified filters (iOS only)          | On / Off
AdGuard DNS:                           | None / Default / Family Protection
Stealth mode options (Windows only)    | ?
Helpdesk ID (if exists):               | ?

[//]: # (This template is meant for missed ad/false positive reports, for other type of reports edit it accordingly)
[//]: # (If this is a crash report, include the crashlog with https://gist.github.com/)

### CODE REVIEW

File: Chinese video ad filtering [hunk 1]
```diff
@@ -0,0 +1,147 @@
+! Version: 202105021
+! Expires: 2 days
+! Homepage: http://bbs.kafan.cn/thread-1866845-1-1.html
+
+v.youku.com##.control-phonewatch,#right-title-ad-banner
+youku.com##+js(aeld,dblclick,ykplayer.switchFullscreen)
+!v.youku.com##+js(nano-stb)
```
Comment (piquark6046) [inline Chinese video ad filtering:7]: These rules are not supported on AdGuard.
Please use `#%#` or `#%#//scriptlet()`.

If you want to get more information, please read [this document](https://kb.adguard.com/en/general/how-to-create-your-own-ad-filters).

File: Chinese video ad filtering [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+||atm.youku.com/
+||cibntv.net/youku/$media
+||valipl.cp31.ott.cibntv.net/$media
+|https://js.ykimg.com/youku/dist/js/lib/ikuAdapterNew.js
+||sealine.youku.com/yk/*callback=adpagesdk$script
+
+iqiyi.com##div[data-adzone],#block-V
+!iqiyi.com##+js(nano-sib)
+iqiyi.com##+js(set, Object.prototype.parseXML, noopFunc)
```
Comment (piquark6046) [inline Chinese video ad filtering:16]: These rules are not supported on AdGuard.
Please use `#%#` or `#%#//scriptlet()`.

If you want to get more information, please read [this document](https://kb.adguard.com/en/general/how-to-create-your-own-ad-filters).

File: Chinese video ad filtering [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+||static.iqiyi.com/js/common/mars_v.js$script
+||iqiyi.com/videos/other/20$xmlhttprequest,object-subrequest
+||iqiyipic.com/image/20*_100000$image
+||ssports.iqiyi.com/json/shop/shopInfo
+show.ssports.com
+||ssports.iqiyi.com/app/$object-subrequest
+
+qq.com##.mod_ad, .txp_ad
+v.qq.com,sports.qq.com##+js(json-prune,ad,vinfo)
+v.qq.com,sports.qq.com##+js(nano-sib, \u5012\u8ba1\u65f6)
```
Comment (piquark6046) [inline Chinese video ad filtering:27]: These rules are not supported on AdGuard.
Please use `#%#` or `#%#//scriptlet()`.

If you want to get more information, please read [this document](https://kb.adguard.com/en/general/how-to-create-your-own-ad-filters).

File: Chinese video ad filtering [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+
+bilibili.com##.gg-floor-module,#slide_ad,#slide_ad+a[data-target-url]
+||api.bilibili.com/x/web-show/res/loc?pf=$~script
+||hdslb.com/bfs/sycp/creative_img/$image
+
+acfun.cn##.pause-display-container,#ad-comment,#ad-player-view
+||aixifan.com/static/*/appGuide/
+
+cctv.com###bgAd_div
+cctv.com,ddrk.me##+js(aeld,dblclick,isImmediatePropagationStopped)
```
Comment (piquark6046) [inline Chinese video ad filtering:45]: This rule is not supported on AdGuard.
Please use `#%#` or `#%#//scriptlet()`.

If you want to get more information, please read [this document](https://kb.adguard.com/en/general/how-to-create-your-own-ad-filters).

File: Chinese video ad filtering [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+huya.com###hy-ad,#hy-ab,#huya-ab
+||g.huya.com/
+
+||pub.funshion.com
+
+*/preroll/$media,domain=91porn.com
+||91p20.space^
+||waust.at^
+
+dililitv.cc,lzvod.net##+js(aeld,dblclick,dp.fullScreen.request)
```
Comment (piquark6046) [inline Chinese video ad filtering:105]: This rule is not supported on AdGuard.
Please use `#%#` or `#%#//scriptlet()`.

If you want to get more information, please read [this document](https://kb.adguard.com/en/general/how-to-create-your-own-ad-filters).

File: Chinese video ad filtering [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+||www.dililitv.cc/*.php?ad=$script
+js.dilidd.com
+dililitv.cc,hmtv.me,imeiju.io##+js(acis,Object.defineProperty,location.href)
```
Comment (piquark6046) [inline Chinese video ad filtering:108]: This rule is not supported on AdGuard.
Please use `#%#` or `#%#//scriptlet()`.

If you want to get more information, please read [this document](https://kb.adguard.com/en/general/how-to-create-your-own-ad-filters).

File: Chinese video ad filtering [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+
+dyjihe.com##.tempWrap
+
+haitur.com,haicaoys.com##.dplayer-web-fullscreen-fix .float-box
+haitur.com##.container:style(max-width:90%!important;width:90%!important)
+haitur.com##.left-content:style(width:77%!important)
+
+imomoe.*,xiamov.com,xxdm8.com##+js(ra, target, base[target])
```
Comment (piquark6046) [inline Chinese video ad filtering:116]: This rule is not supported on AdGuard.
Please use `#%#` or `#%#//scriptlet()`.

If you want to get more information, please read [this document](https://kb.adguard.com/en/general/how-to-create-your-own-ad-filters).

File: Chinese video ad filtering [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+dyjihe.com,haicaoys.com,haitur.com,imomoe.*,kakadm.com,milimili.tv,qiqidongman.com,vodxc.in##+js(ra, target, a[href^="/"])
```
Comment (piquark6046) [inline Chinese video ad filtering:117]: This rule is not supported on AdGuard.
Please use `#%#` or `#%#//scriptlet()`.

If you want to get more information, please read [this document](https://kb.adguard.com/en/general/how-to-create-your-own-ad-filters).

File: Chinese video ad filtering [hunk 9]
```diff
@@ -0,0 +1,147 @@
...
+||alicdn.com/$image,domain=api.xiaomingming.org
+imomoe.*###adl,#adr
+
+yatu.tv##.dplayer-web-fullscreen-fix #tburl,#loadtop,[id^=myas],#myds1,#daohang
+yatu.tv##+js(set,document.oncontextmenu,null)
```
Comment (piquark6046) [inline Chinese video ad filtering:122]: This rule is not supported on AdGuard.
Please use `#%#` or `#%#//scriptlet()`.

If you want to get more information, please read [this document](https://kb.adguard.com/en/general/how-to-create-your-own-ad-filters).

File: Chinese video ad filtering [hunk 10]
```diff
@@ -0,0 +1,147 @@
...
+||tracker.$xhr,websocket,script,ping
```
Comment (piquark6046) [inline Chinese video ad filtering:123]: `$xhr` is not support on AdGuard.

**Review by ghost — CHANGES_REQUESTED** [review]
There are four issues on your PR.

* Cannot connect their homepage, so I cannot see the license of these codes.
timeout
time


* I and AdGuard filter managers need reproducible URLs to verify your code.

* Finally, I think It will be good for what you contribute at `ChineseFilter/sections/`.

File: Chinese video ad filtering (hunk 10 above)
Comment (Alex-302) [reply Chinese video ad filtering:123]: supported, but better use the full modifier

**Pull request closed without merging**